### PR TITLE
Add nuspec for NEST project

### PR DESCRIPTION
**Pull request by billbogaiv**
Can't get the package without a nuspec!

Ref. https://github.com/ritterim/filter/issues/30

### CODE REVIEW

File: src/Filter.Nest/Filter.Nest.nuspec [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <metadata>
+    <id>RimDev.Filter.Nest</id>
+    <version>0.0.0</version>
+    <authors>Ritter Insurance Marketing</authors>
+    <licenseUrl>https://raw.githubusercontent.com/ritterim/filter/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/ritterim/filter</projectUrl>
+    <description>The "classy" way to filter collections. Supports NEST.</description>
+    <copyright>Copyright 2016 Ritter Insurance Marketing</copyright>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <tags>rimdev ienumerable filter range</tags>
```
Comment (kendaleiv) [inline src/Filter.Nest/Filter.Nest.nuspec]: Want to add a `nest` tag?

File: src/Filter.Nest/Filter.Nest.nuspec [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  </metadata>
+  <files>
+    <file src="bin\Release\RimDev.*.dll" target="lib\net45" />
```
Comment (kendaleiv) [inline src/Filter.Nest/Filter.Nest.nuspec]: Is this correct?

File: src/Filter.Nest/Filter.Nest.nuspec (hunk 1 above)
Comment (billbogaiv) [inline src/Filter.Nest/Filter.Nest.nuspec]: Added.

File: src/Filter.Nest/Filter.Nest.nuspec (hunk 2 above)
Comment (billbogaiv) [inline src/Filter.Nest/Filter.Nest.nuspec]: Yes. The single wildcard does not use a period as a delimiter.

**Pull request merged**